src/main/resources/enchantCraftingList.json

```diff
@@ -0,0 +1,169 @@
+{

```

Comment:
Can we change "Normal" to "normal", "Other" to "other", and "Material" to "material". Let's also rename "Items" to "craftableItems" to be a bit more specific.

It should be fairly quick with a find & replace all function on any IDE and will require multiple file changes.